en/airframes/adding_a_new_frame.md

```diff
@@ -115,12 +115,14 @@ S: 0 1   6500   6500      0 -10000  10000
 
 Where each number from left to right means:
```

Comment:
@barzanisar At the top of this section is a line: "A typical mixer file is shown below (for general information about mixing see: [Concepts > Mixing](../concept/mixing.md)). "

Suggest you change this to (something like):

```
> **Note** First read [Concepts > Mixing](../concept/mixing.md). This provides background information required to interpret this mixer file.

A typical mixer file is shown below.
```

You will note that I just deleted a bunch of my own comments asking for more clarification ... because I didn't read the concepts article first :-)

Reply:
I see that is done. Thanks.